spring-boot-project/spring-boot-tools/spring-boot-maven-plugin/src/main/java/org/springframework/boot/maven/ArtifactsLibraries.java

```diff
@@ -50,12 +50,7 @@ public class ArtifactsLibraries implements Libraries {
 	private static final Map<String, LibraryScope> SCOPES;
 
 	static {
-		Map<String, LibraryScope> libraryScopes = new HashMap<>();
-		libraryScopes.put(Artifact.SCOPE_COMPILE, LibraryScope.COMPILE);
-		libraryScopes.put(Artifact.SCOPE_RUNTIME, LibraryScope.RUNTIME);
-		libraryScopes.put(Artifact.SCOPE_PROVIDED, LibraryScope.PROVIDED);
-		libraryScopes.put(Artifact.SCOPE_SYSTEM, LibraryScope.PROVIDED);
-		SCOPES = Collections.unmodifiableMap(libraryScopes);
+		SCOPES = Map.of(Artifact.SCOPE_COMPILE, LibraryScope.COMPILE, Artifact.SCOPE_RUNTIME, LibraryScope.RUNTIME, Artifact.SCOPE_PROVIDED, LibraryScope.PROVIDED, Artifact.SCOPE_SYSTEM, LibraryScope.PROVIDED);
```

Comment:
Like https://github.com/spring-projects/spring-boot/pull/38352, I prefer this as currently written as I think it's easier to read. Please revert.